sklearn/neural_network/tests/test_mlp.py

```diff
@@ -175,8 +175,9 @@ def test_gradient():
     # are correct. The numerical and analytical computation of the gradient
     # should be close.
     for n_labels in [2, 3]:
-        n_samples = 5
+        n_samples = 100
         n_features = 10
+        np.random.seed(42)
```

Comment:
This is usually not the way we do it.
To avoid changing the global random state, we prefer using:
```
rng = np.random.RandomState(42)
X = rng.random((n_samples, n_features))
```

Reply:
Ah! Makes sense. Will make the changes ✌️

Reply:
@TomDLT Made the changes